src/components/Filters/FiltersMobile.tsx

```diff
@@ -121,9 +121,27 @@ export const FiltersMobile = ({
     toggleDrawer();
   };
 
+  const handleKeyDown = (event: React.KeyboardEvent<HTMLInputElement>) => {
+    const { key } = event;
+
+    if (
```

Comment:
lets create a helper fo this logic. Something like function checkKeydownEvent which will return a bool and use it in the places, where you need it